### PR TITLE
Html fixes from metalsmith migration

### PR DESCRIPTION
**Pull request by crwallace**

### CODE REVIEW

File: content/includes/1-topic-landing.html [hunk 1]
```diff
@@ -1,23 +1,26 @@
 <div class="main" role="main">
 {% unless page.template == 'alt-1-topic-landing' %}
 
-  {% if collection == 'disability-benefits' %}
-    {% include action-bar.html action_text="Apply for Disability Benefits" action_url="/disability-benefits/apply-for-benefits/" action_class="va-action-bar--header" %}
+  {% if path contains 'disability-benefits' %}
```
Comment (awong-dev) [inline content/includes/1-topic-landing.html]: Is this going to cause a problem with substrings?

Like  `/disability-benefits-exceptions/` would match this as well...

Comment (crwallace) [inline content/includes/1-topic-landing.html]: Where does `/disability-benefits-exceptions/` appear?

**Comment by awong-dev** [comment]
I skimmed this but realisticaly this is going to be a rubber-stamp LGTM.

Go ahead and merge.

File: content/includes/1-topic-landing.html (hunk 1 above)
Comment (awong-dev) [inline content/includes/1-topic-landing.html]: It doesn't. But you're sorta leaving a timebomb for future maintainers.

**Pull request merged**